packages/waku/src/lib/rsc/handler-prd.ts

```diff
@@ -42,7 +42,7 @@ export function createHandler<
   return async (req, res, next) => {
     const config = await configPromise;
     const basePrefix = config.basePath + config.rscPath + '/';
-    const pathStr = req.url.slice(new URL(req.url).origin.length);
+    const pathStr = new URL(req.url).pathname;
```

Comment:
This isn't our design. `pathStr` means `pathname` + `query`. We don't have examples that use query yet, but we can't drop it here.